jest.config.js

```diff
@@ -7,7 +7,7 @@ const config = require('./.config/jest.config');
 module.exports = {
   // Jest configuration provided by Grafana scaffolding
   ...config,
-  setupFilesAfterEnv: ['@testing-library/jest-dom/extend-expect', ...(config.setupFilesAfterEnv || [])],
+  setupFilesAfterEnv: ['<rootDir>/src/test/jest-setup.tsx'],
```

Comment:
I wanted to simplify the jest set-up as it was quite confusing all the places you have to look to find out what our jest dependencies and mocks are. I'm starting to consolidate all the ones we manage to the jest-setup file, rather than having them spread between here, the various mocks files, and the jest setup file.